app/src/editor/canvas/index.jsx

```diff
@@ -552,22 +553,25 @@ const CanvasEditWrap = () => {
 
 const CanvasErrorPage = () => (
     <ErrorPageContent>
-        <Link
+        <Button
+            type="special"
```

Comment:
This maybe part of the other PR but should `type` be called something else since it's a DOM attribute? It get's repurposed in `Button` so you can't set it manually.